manage/time-accounting.rst

```diff
@@ -2,30 +2,34 @@ Time Accounting
 ***************
 
 If you want to know how much time you need for your each project or ticket,
-enable time accounting (turn on the switch on the top left side of the page).
+enable the time accounting (turn on the switch on the top left side of the
+page).
 
 .. figure:: /images/manage/time-accounting/time-accounting-management.png
-   :alt: Time accounting management page in Zammad
+   :alt: Time Accounting Management Screen in Zammad
 
 How it works
 ------------
 
-Zammad's time accounting uses ticket selectors (filters) to check if a ticket
-is applicable for time accounting or not. If a ticket applies for accounting,
-Zammad will request the agent to provide how much time was needed to process
-the current ticket step.
+Settings
+~~~~~~~~
+
+Zammad's time accounting uses ticket **Selector** (filter) to check if a ticket
+is considered for the time accounting or not. If a ticket is applicable, Zammad
+will request the agent to provide how much time was needed to process the
```

Comment:
`many` instead of `much`? 🤔

Reply:
I disagree: the `time` noun is not countable, so it's always considered as singular. Therefore, we describe it with `much`, while `many` is reserved to countable nouns in plural form (e.g. `items`, `tasks`, etc).

Reply:
Got it, thanks.